l10n_it_fatturapa/models/company.py

```diff
@@ -49,6 +50,22 @@ class ResCompany(models.Model):
              'residente, con stabile organizzazione in Italia'
         )
 
+    @api.multi
+    @api.constrains(
+        'fatturapa_sequence_id'
+    )
+    def _check_fatturapa_sequence_id(self):
+        for company in self:
+            if company.fatturapa_sequence_id:
+                journals = self.env['account.journal'].search([
+                    ('sequence_id', '=', company.fatturapa_sequence_id.id)
```

Comment:
aggiungerei `limit=1` alla `search`

Reply:
and I'd replace `journals` by `journal` everywhere in this method.

Reply:
A questo punto rinominerei `journals` con `journal` dappertutto all'interno di questo metodo.